lib/subjects/postgresql_selection.rb

```diff
@@ -17,6 +17,15 @@ def select(options={})
       results.take(limit)
     end
 
+    def any_workflow_data(options={})
+      @opts = options
```

Comment:
going to rework as explicit method params and remove the state...reading the specs their use case is to modify the behaviour of the method at runtime.